### ISSUE TITLE
how to change the learning rate

### ISSUE THREAD

**Issue by kingrain213** [hx173149/C3D-tensorflow#38]
I have trained my own model use your code, thank you for your share.
now the rate is waving only in a little scale, but I havent got the learning rate parameter，
could you tell me how can I change the learning rate ?
what is your original learning rate parameter in your train?

**Comment by zzy123abc** [comment]
You can use sgd to change the learning rate.
This code is adam,so the learning rate is waving only in a little scale.

**Comment by jackietom** [comment]
I think you can change the learning rate by changing the learning rate of opt_stable and opt_finetuning in the train_c3d_....py file. In detail, you can change the number in " opt_stable = tf.train.AdamOptimizer(1e-6)" and  " opt_finetuning = tf.train.AdamOptimizer(1e-3)"
